lib/reporters/base.js

```diff
@@ -31,10 +31,12 @@ var isatty = tty.isatty(1) && tty.isatty(2);
 exports = module.exports = Base;
 
 /**
- * Enable coloring by default.
+ * Enable coloring by default, except in the browser interface.
  */
 
-exports.useColors = supportsColor || (process.env.MOCHA_COLORS !== undefined);
+exports.useColors = process.env
```

Comment:
If `supportsColor` is false, it falls through to the next check, and in the browser `process.env` is undefined so `process.env.MOCHA_COLORS` fails.